src_c/line.c

```diff
@@ -49,16 +57,178 @@ pg_line_init(pgLineObject *self, PyObject *args, PyObject *kwds)
     return 0;
 }
 
+static PyObject *
+pgLine_New(pgLineBase *l)
+{
+    return _pg_line_subtype_new4(&pgLine_Type, l->ax, l->ay, l->bx, l->by);
+}
+
 static PyObject *
 pg_line_copy(pgLineObject *self, PyObject *_null)
 {
     return _pg_line_subtype_new4(Py_TYPE(self), self->line.ax, self->line.ay,
                                  self->line.bx, self->line.by);
 }
 
+static PyObject *
+pg_line_update(pgLineObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    if (!pgLine_FromObjectFastcall(args, nargs, &(self->line))) {
```

Comment:
Parenthesis here are not needed
```suggestion
    if (!pgLine_FromObjectFastcall(args, nargs, &self->line)) {
```